From: Phil Elwell <phil@raspberrypi.org>
Date: Thu, 28 Nov 2019 15:49:08 +0000
Subject: arm/dts: 2711: Add 'pcie0' alias

It is useful for the firmware to be able to locate the pcie DT node,
so add an alias pointing to it in the same way that "ethernet0"
points to the genet.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
index 6ce9a58a6ed5..a967a7b86f2c 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -30,6 +30,7 @@
 		/delete-property/ ethernet;
 		/delete-property/ intc;
 		ethernet0 = &genet;
+		pcie0 = &pcie_0;
 	};
 };
 
